File: 2545_sort_the_students_by_their_kth_score/Solution.cs
// LeetCode 2545 - Sort the Students by Their Kth Score
// https://leetcode.com/problems/sort-the-students-by-their-kth-score/

using System;

public class Solution {
    public int[][] SortTheStudents(int[][] score, int k) {
        Array.Sort(score, (a, b) => b[k].CompareTo(a[k]));
        return score;
    }
}

File: 2545_sort_the_students_by_their_kth_score/solution.py
# LeetCode 2545 - Sort the Students by Their Kth Score
# https://leetcode.com/problems/sort-the-students-by-their-kth-score/

from typing import List


class Solution:
    def sortTheStudents(self, score: List[List[int]], k: int) -> List[List[int]]:
        score.sort(key=lambda row: -row[k])
        return score
